data/events.txt

```diff
@@ -1176,29 +1176,23 @@ event "navy using mark ii ships"
 			"Rainmaker (Mark II)" 2
 			"Gunboat (Mark II)"
 
-fleet "gunboat only"
-	government "Republic"
-	names "republic small"
-	cargo 0
-	personality
-		heroic
-	variant
-		"Gunboat (Mark II)"
-
 event "gunboats in Alnasl"
-	system Alnasl
+	system "Alnasl"
 		fleet "Small Southern Merchants" 1300
 		fleet "Large Southern Merchants" 2500
-		fleet "gunboat only" 800
```

Comment:
Are you intending to remove this "gunboat only" fleet? This change will mean that there won't be any ongoing republic fleet spawns in this system. Might it be good to leave the gunboat spawn in, although potentially lowering the spawn rate a touch to avoid stacking too many on the player might be good.

Reply:
Looks good now.